### PR TITLE
Patch/additional validation acquirer steps

### PR DESCRIPTION
**Pull request by AShaww**
Implemented additional use cases for the newly implemented validation change with express-validator.

Added additional cases to most radio pages/steps excluding legal-power and legal-review 

A list of effected pages/steps that need to have a `radio selected`:

data-access,
data-travel,
role,
delivery,
format,
security-review,

we now see an error as such:
image

for each of the pages/steps mentioned

### CODE REVIEW

File: src/views/acquirer/format.njk [hunk 1]
```diff
@@ -3,6 +3,10 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
+{% if errorMessage %}
```
Comment (duncanheron) [inline src/views/acquirer/format.njk:6]: This is repeated, we should be able to use nunjucks to deal with this across all the templates

**Comment by kjoshi** [comment]
I think the validation should only run when `Save and Continue` is pressed.
Users should still be allowed to go back to the overview page without answering by pressing `Save and Return`.

My understanding was that `Save and Continue` means "I've though about my answer and I'm finished with this step, let me carry on to the next one" and `Save and Return` means "I still need to think about this". So I don't think we need to be validating an answer/selection until it's final.

**Comment by AShaww** [comment]
I have made them changes, return button skips validation.

removed repeated code, let Nunjucks handle the errorMessage as @duncanheron suggested

**Comment by kjoshi** [comment]
This seems to work well, thanks Amir.
I don't see any reason why we can't also do the same thing for the rest of the radio buttons, e.g. `data-type` and the legal ones.

I think it makes sense to only even let someone `Save and Continue` if the've answered a question.
The `Save and Return` button is there to give people a chance to think before committing to an answer...

**Review by madz-lw — APPROVED** [review]
LGTM thanks Amir 👍

**Pull request merged**